### PR TITLE
feat(ui): Add entity names to some toast messages

### LINKED ISSUE #552: Show entity name in delete confirmation "toast"
After deleting e.g. an organization, a "toast" scrolls in at the bottom right that confirms the deletion. However, it only contains a generic message. We should name the entity that was deleted as part of the confirmation.

### PR DESCRIPTION
**Pull request by Etsija**
Please see the commit message for details.

### CODE REVIEW

File: ui/src/routes/_layout/create-organization.tsx [hunk 1]
```diff
@@ -58,7 +58,10 @@ const CreateOrganizationPage = () => {
     onSuccess() {
       toast({
         title: 'Create Organization',
-        description: 'New organization created successfully.',
+        description:
+          'New organization "' +
+          form.getValues().name +
+          '" created successfully.',
```
Comment (sschuberth) [inline ui/src/routes/_layout/create-organization.tsx:64]: Just wondering, can / should [template literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals) be used here to improve readability via string interpolation? Like

    `New organization "${form.getValues().name}" created successfully.`

Comment (Etsija) [reply ui/src/routes/_layout/create-organization.tsx:64]: I've never been a fan of those, as I'm not seeing why they would be more readable.

Comment (sschuberth) [reply ui/src/routes/_layout/create-organization.tsx:64]: We should have a clear line here, and do it (or not) consistently, if at all. I'm curious about @mmurto's advice about best practices here.

Comment (Etsija) [reply ui/src/routes/_layout/create-organization.tsx:64]: As I'm sure he's pro-literals, I've already made the changes.

**Review by sschuberth — DISMISSED** [review]

**Comment by sschuberth** [comment]
> Resolves #552.

Actually this bit should go to the commit message, not in the PR discussion.

File: ui/src/routes/_layout/organizations/$orgId/create-product.tsx [hunk 2]
```diff
@@ -60,7 +60,7 @@ const CreateProductPage = () => {
     onSuccess() {
       toast({
         title: 'Create Product',
-        description: 'New product created successfully.',
+        description: `New product "${form.getValues().name}" created successfully.`,
```
Comment (mmurto) [inline ui/src/routes/_layout/organizations/$orgId/create-product.tsx:63]: I would change all of these to use the return value of the mutations to get the names/etc. from there. I believe the toast will show a wrong name with the current implementation if someone clicks submit and writes something to the input while the submit is processing. The return value should be available to the onSuccess callback.

**Review by mmurto — CHANGES_REQUESTED** [review]

File: ui/src/routes/_layout/organizations/$orgId/create-product.tsx (hunk 2 above)
Comment (sschuberth) [reply ui/src/routes/_layout/organizations/$orgId/create-product.tsx:63]: No, I've removed this PR from the merge queue, so you can still fix it.

**Review by mmurto — APPROVED** [review]

**Pull request merged**